streams/src/main/java/org/apache/kafka/streams/state/internals/ExceptionUtils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.state.internals;
```

Comment:
I just plunked this class where it was needed right now; we can move it later if we want to use it elsewhere.

Reply:
Maybe we should a new package `org.apache.kafka.streams.internals` ?

Reply:
Yeah, something like that sounds good. Still, I'd like to select the right location after we need to use it from two or more different packages.